avm/res/network/network-manager/network-group/main.bicep

```diff
@@ -12,10 +12,10 @@ param name string
 
 @maxLength(500)
 @sys.description('Optional. A description of the network group.')
-param description string = ''
+param description string?
 
 @sys.description('Optional. Static Members to create for the network group. Contains virtual networks to add to the network group.')
-param staticMembers array = []
+param staticMembers staticMembersType?
```

Comment:
Only either the parameter or the type must be nullable

Reply:
Updated